src/app/landing/landing.component.scss

```
@@ -0,0 +1,201 @@
+
+.slogan {
+  display: flex;
+  align-items: flex-start;
+  gap: 0.75rem;
+  font-size: 0.95rem;
+  color: #333;
+  font-weight: 500;
+}
+
+.slogan span {
+  word-wrap: break-word;
```

Stylelint (17.6.0)

[error] Expected "word-wrap" to be "overflow-wrap" (property-no-deprecated)

(property-no-deprecated)